ansible/roles/graphite/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+
+- name: "Build hosts file"
+  sudo: yes
+  lineinfile:
+    dest: /etc/hosts
+    regexp: ".*{{ fqdn }}$"
+    line: "127.0.1.1 {{ fqdn }}"
+    state: present
+
+- name: Set Hostname with hostname command
+  sudo: yes
+  hostname: name="{{ fqdn }}"
+
+- name: update apt cache
+  apt:
+    update_cache: yes
+  sudo: yes
+
+- name: install ssl system requirements
+  sudo: yes
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items: ssl_requirements
+  tags:
+    - packages
+
+- name: install system packages
+  sudo: yes
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items: system_packages
+  tags:
+    - packages
+
+- command: cp /usr/share/graphite-web/graphite.wsgi /usr/lib/python2.7/dist-packages/graphite/graphite_web.py
+  args:
+    creates: "/usr/lib/python2.7/dist-packages/graphite/graphite_web.py"
+  sudo: true
+
+- include: carbon.yml
+
+- include: systemd.yml
+  tags:
+    - systemd
+
+- include: postgresql.yml
+  tags:
+    - postgresql
+
+- name: ensure graphite is running
+  sudo: true
+  service:
+    name: graphite
+    state: restarted
```

Comment:
Do you want ``started`` here instead of ``restarted``?  Maybe it doesn't matter?